assets/js/components/ClustersList.jsx

```diff
@@ -74,7 +74,9 @@ const ClustersList = () => {
         key: 'name',
         filter: true,
         render: (content, item) => (
-          <ClusterLink cluster={item}>{content}</ClusterLink>
+          <ClusterLink cluster={item}>
+            <div className="tn-clustername">{content}</div>
+          </ClusterLink>
```

Comment:
I think we can't do that this way, try adding the `className` prop directly to the `ClusterLink` component, propagating it to the `span` underneath

Reply:
... or you could just convert the `div` to a `span`. Your call.

Reply:
I had already tried the first approach and it didn't work, going for the second one